### ISSUE TITLE
"region" is deprecated in google_container_cluster

### ISSUE THREAD

**Issue by czka** [terraform-google-modules/terraform-google-kubernetes-engine#41]
```
$ terraform --version
Terraform v0.11.11
+ provider.google v1.19.1
```

You guys require the `region` setting in your module:
```
$ terraform plan
Error: module "gke": missing required argument "region"
```

But:
```
Warning: module.gke.google_container_cluster.primary: "region": [DEPRECATED] This field is in beta and will be removed from this provider. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/provider_versions.html for more details.
```

**Comment by czka** [comment]
Or am I supposed to use this module with a beta Google provider only? I can't see this clearly stated in the README.md.

**Comment by aaron-lane** [comment]
Hi @czka.

I apologize for the delay in responding to your issue!

Presently, the module requires both the Google provider v2.3 and the Google Beta provider v2.3. We are working to address field deprecations but they should not affect your usage of the module.

**Comment by arvindpai** [comment]
even zone seems to be deprecated. As stated in terraform.io for container cluster.
![Screenshot region-zone](https://user-images.githubusercontent.com/17788117/57389320-c65f8200-716e-11e9-891d-5697241de8c3.png)

**Comment by morgante** [comment]
This is captured in #147 as part of a larger effort to consolidate resource support.

**Comment by aaron-lane** [comment]
This was fixed by #223.
